src/components/organisms/About/About.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "../About/About.css";
+
+
+interface AboutImageData{
+  path: string;
+  alt: string;
+}
+
+interface AboutProps {
+  title: string;
+  subtitle: string;
+  content: string;
+  images: Array<AboutImageData>;
+}
+
+const About: React.FC<AboutProps> = (props) => {
```

Comment:
Optional, we should avoid using React.FC https://github.com/facebook/create-react-app/pull/8177

Reply:
Changed, hope it's correct